src/govuk/tools/_compatibility.scss

```diff
@@ -34,3 +36,14 @@
     @error "Non existent product '#{$product}'";
   }
 }
+
+/// Temporary public version of _govuk-compatibility to suppress local warnings
+///
+/// @access public
+
+@mixin govuk-compatibility($product) {
+  @warn "govuk-compatibility is deprecated. Support for legacy codebases will be dropped in v5.0";
```

Comment:
Because the tests still call `govuk-compatibility` we now have warnings in our test output.

We've handled in the past by mocking Sass's `@warn` function, which also allows us to add tests that 'expect' the warning, which means we can be confident it's working as intended.

(We could also update the tests to call the private version of the mixin, **but** then we're not testing the public bit of the API and if we made a mistake in this wrapper function we might not spot it)

See https://github.com/alphagov/govuk-frontend/pull/2156 for an example.